drivers/include/lc709203f.h

```diff
@@ -0,0 +1,391 @@
+
+/**
+ * @brief  reads battery voltage from Sensor
+ *
+ * @param[in] *dev      pointer to lc709203f_t struct containing the i2c device and the address
+ *
+ * @return              voltage in mV
+ */
+int16_t lc709203f_get_voltage(const lc709203f_t *dev);
+
+/**
+ * @brief  reads battery state of charge from Sensor
+ *
+ * @param[in] *dev      pointer to lc709203f_t struct containing the i2c device and the address
+ *
+ * @return              rsoc in %
+ */
+int16_t lc709203f_get_rsoc(const lc709203f_t *dev);
+
+/**
+ * @brief  reads battery indicator to empty from Sensor
+ *
+ * @param[in] *dev      pointer to lc709203f_t struct containing the i2c device and the address
+ *
+ * @return              ite in 0.1%
+ */
+int16_t lc709203f_get_ite(const lc709203f_t *dev);
+
+/**
+ * @brief  reads Sensor id
+ *
+ * @param[in] *dev      pointer to lc709203f_t struct containing the i2c device and the address
+ *
+ * @return              id value
+ */
+int16_t lc709203f_get_id(const lc709203f_t *dev);
+
+/**
+ * @brief  reads cell temperature from sensor
+ *
+ * @param[in] *dev      pointer to lc709203f_t struct containing the i2c device and the address
+ *
+ * @return              temp in 0.1°C steps
+ */
+int16_t lc709203f_get_cell_temp(const lc709203f_t *dev);
+
+/**
+ * @brief  reads status bit register (Temperature obtaining mode)
+ *
+ * @param[in] *dev      pointer to lc709203f_t struct containing the i2c device and the address
+ *
+ * @return              THERMISTOR_MODE Thermistor mode
+ * @return              I2C_MODE I2C Mode
+ */
+lc709203f_temp_obtaining_mode_t lc709203f_get_status_bit(const lc709203f_t *dev);
+
+/**
+ * @brief  reads IC Power Mode register
+ *
+ * @param[in] *dev      pointer to lc709203f_t struct containing the i2c device and the address
+ *
+ * @return              OPERATIONAL_MODE Operational Mode
+ * @return              SLEEP_MODE Sleep mode
+ */
+lc709203f_power_mode_t lc709203f_get_power_mode(const lc709203f_t *dev);
+
+/**
+ * @brief  reads Alarm Low Cell Voltage Register
+ *
+ * @param[in] *dev      pointer to lc709203f_t struct containing the i2c device and the address
+ *
+ * @return              0 Disable
+ * @return              Threshold value in 1mV steps
+ */
+int16_t lc709203f_get_alarm_low_voltage(const lc709203f_t *dev);
+
+/**
+ * @brief  reads Alarm Low RSOC Register
+ *
+ * @param[in] *dev      pointer to lc709203f_t struct containing the i2c device and the address
+ *
+ * @return              0 Disable
+ * @return              Threshold value in %
+ */
+int16_t lc709203f_get_alarm_low_rsoc(const lc709203f_t *dev);
+
+/**
+ * @brief  reads Change of the Parameter register, Battery Profile selection
+ *
+ * @param[in] *dev      pointer to lc709203f_t struct containing the i2c device and the address
+ *
+ * @return              Battery profile number (1 or 0)
+ */
+int16_t lc709203f_get_change_of_parameter(const lc709203f_t *dev);
+
+/**
+ * @brief  readsAPT (Adjustment Pack Thermistor)
```

Comment:
missing space

Reply:
Where would i need to add a space here? Like this: `lc709203f_get_change_of_parameter( const lc709203f_t *dev )`? I couldn't find anything in the coding convention.

Reply:
the github comments always refer to the last line one can see in the "code context thingy". What i meant was `readsAPT` in the brief line of the doc ;)